css/properties/custom-property.json

```diff
@@ -142,6 +142,39 @@
               "deprecated": false
             }
           }
+        },
+        "env": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/env",
```

Comment:
this url is not existant.

Reply:
I know but I just want to point that out :)

Reply:
I've created a first version of the page, it has to be completed: https://developer.mozilla.org/en-US/docs/Web/CSS/env